osu.Game/Screens/Edit/Compose/Components/SelectionHandler.cs

```diff
@@ -33,10 +33,14 @@ namespace osu.Game.Screens.Edit.Compose.Components
     /// </summary>
     public class SelectionHandler : CompositeDrawable, IKeyBindingHandler<PlatformAction>, IHasContextMenu
     {
+        /// <summary>
+        /// The currently selected blueprints.
+        /// Should be used when operations are dealing directly with the visible blueprints.
+        /// For more general selection operations, use EditorBeatmap.SelectedHitObjects instead.
```

Comment:
not a huge fan of having both this and `EditorBeatmap.SelectedHitObjects`, but i can't see a better way to escape this...

took me a second to understand the xmldoc as well, but also not sure how to word it better.

Reply:
There's a few cases where it seems like using this is what we want, but agree that if we can make it non-public at least, it would be an improvement.

Reply:
doesn't seem too possible given that a few places actually want to be using the drawable selected blueprint rather than the raw HOs... think I'm fine with this as is.

as an aside the xmldoc reference can be done with a fully qualified name:

```suggestion
        /// Should be used when operations are dealing directly with the visible blueprints.
        /// For more general selection operations, use <see cref="osu.Game.Screens.Edit.EditorBeatmap.SelectedHitObjects"/> instead.
```

but it's a bit ghastly...